extensions/cryptography.py

```
@@ -6,7 +6,7 @@
 from __future__ import annotations
 
 # HANDLER -> LOAD
-from website import *
+from abscissa import *
```

Ruff (0.12.2)

`from abscissa import *` used; unable to detect undefined names

(F403)